### DISCUSSION TITLE
On VSCode, commit succeeds when run from the terminal but fails from the panel.

### DISCUSSION THREAD

**Discussion by lukstafi** [typicode/husky#1220] [Q&A]
See [this VSCode extension project](https://github.com/lukstafi/navi-parens). When I run `git commit -m "..."` from the terminal (VSCode on Windows), everything works, but when I click `Commit` on the source control panel, I get:
```
Failed to run tests
husky - pre-commit hook exited with code 1 (error)
```

**Comment by tylerkess** [comment]
Any answer? This started happening to me recently
